Longest_Common_Subsequence/Longest_Common_Subsequence.rb

```diff
@@ -0,0 +1,40 @@
+# Problem Statement- Given two sequences, find 
+# the length of longest subsequence present in both of them. 
+# A subsequence is a sequence that can be derived 
+# from another sequence by deleting some or 
+# no elements without changing the order of the remaining elements.
+
+def longest_common_subsequence(arr1, arr2, size1, size2) 
+    lcs = Array.new(size1 + 1){Array.new(size2 + 1)}
+    
+    for i in 0..size1
+        for j in 0..size2
+            if (i == 0 || j == 0) 
+                lcs[i][j] = 0    
+            # when the last character of both subsequences match, 
+            # increase length of lcs by 1
+            elsif (arr1[i - 1] == arr2[j - 1]) 
+                lcs[i][j] = lcs[i - 1][j - 1] + 1
+            # when the last character is not same, 
+            # take maximum obtained by adding
+            # one character to one of the subsequences 
+            else
+                lcs[i][j] = [lcs[i - 1][j], lcs[i][j - 1]].max
+            end
+        end 
+    end
+    
+    lcs[size1][size2]
+end 
+
+# Driver Code
+arr1 = [10, 15, 20, 25, 30, 35, 40]
```

Comment:
It is possible to take dynamic inputs in Ruby. Please update the code to take dynamic inputs. This requirement is only relaxed for languages like js or php where we have to create forms for dynamic inputs.

Reply:
Have changed to dynamic input